backend/app/adapter/gqlapi/resolver/authmutation.go

```diff
@@ -49,24 +49,24 @@ func (a AuthMutation) CreateShortLink(args *CreateShortLinkArgs) (*ShortLink, er
 		m  shortlink.ErrMaliciousLongLink
 	)
 	if errors.As(err, &ae) {
-		return nil, ErrAliasExist(shortLink.CustomAlias)
+		return nil, ErrAliasExist(ae.Error())
```

Comment:
I think we we should avoid leaking the exact error to the API consumer to prevent hackers from launching cyber attack. Can we change back to what we had before?

Reply:
I did this to prevent panic from dereferencing null pointer. I'll change it back and add some conditional logic to prevent null dereference.

Reply:
Adding another comment to mention that no need to add more conditional logic, I reused the `GetLongLink` and `GetCustomAlias` methods with "" as `defaultVal`.

Reply:
Will push this out along with my other changes in a bit.